MailLibrary/Drivers/ImapDriver.php

```diff
@@ -487,4 +487,61 @@ protected function encodeMailboxName($name)
 		return mb_convert_encoding($name, 'UTF7-IMAP', 'UTF-8');
 	}
 
+	/**
+	 * Save mail content as eml to given file
+	 * @param int $messageUID
+	 * @throws DriverException
+	 * @return string the mail content
+	 */
+	public function retrieveRawMessage($messageUID)
```

Comment:
This is much better. I didn't realized that I can use `fopen('php://temp', 'w+b');`. Thank you.